Makefile.toml

```diff
@@ -0,0 +1,85 @@
+[config]
+default_to_workspace = false
+
+[tasks.ignore-members]
+workspace = false
+
+[tasks.clean]
+command = "cargo"
+args = ["clean"]
+
+[tasks.build]
+dependencies = ["clean"]
+command = "cargo"
+args = ["build"]
+
+[tasks.test]
+dependencies = ["clean"]
+command = "cargo"
+args = ["test"]
+
+[tasks.format_inst]
+install_crate = { crate_name = "rustfmt", rustup_component_name = "rustfmt", binary = "rustfmt", test_arg = "--help" }
+
+[tasks.format]
+dependencies = ["format_inst"]
+command = "cargo"
+args = ["fmt"]
+
+[tasks.format-check]
+dependencies = ["format_inst"]
+command = "cargo"
+args = ["fmt", "--check"]
+
+[tasks.clippy-inst]
+install_crate = { crate_name = "clippy", rustup_component_name = "clippy", binary = "clippy", test_arg = "--help" }
+
+[tasks.clippy]
+dependencies = ["clippy-inst"]
+command = "cargo"
+args = ["clippy", "--", "--deny", "warnings"]
+
+[tasks.header-add]
+script = { file = "./scripts/add_header_if_missing.sh" }
+args = ["HEADER-APACHE2", "./!(target)/**/*.rs"]
+
+[tasks.header-check]
+env = { CHECK_DIRTY = "true", DRY_RUN = "true" }
+run_task = "header-add"
+
+[tasks.audit-inst]
+command = "cargo"
+args = ["install", "cargo-audit"]
+
+[tasks.audit]
+dependencies = ["audit-inst"]
+command = "cargo"
+args = ["audit"]
+
+[tasks.cov]
+command = "cargo"
+args = ["llvm-cov", "--workspace", "--lcov", "--output-path", "lcov.info"]
+
+[tasks.bench]
+dependencies = ["test"]
+command = "cargo"
+args = ["bench"]
+
+[tasks.udeps-inst]
+command = "cargo"
+args = ["install", "cargo-udeps", "--locked"]
+
+[tasks.udeps]
+dependencies = ["udeps-inst"]
+command = "cargo"
+args = ["udeps", "--all-targets"]
+
+
+
```

Comment:
```suggestion
[tasks.msrv]
command = "cargo"
args = ["msrv"]

[tasks.msrv-check]
command = "cargo"
args = ["hack", "check", "--rust-version", "--workspace", "--ignore-private"]
```

And add them to CI

Reply:
I don't understand the utility of this test and I have removed `rust-version = "1.60"` from `Cargo.toml` for now.